.travis.yml

```diff
@@ -18,8 +18,9 @@ install:
 - npm install
 
 before_script:
-- npm install
+  - npm install
+  - npx webdriver-manager update
```

Comment:
This is not needed because we don't connect to local runners, so this can be removed